### ISSUE TITLE
excelise breaks all styles

### ISSUE THREAD

**Issue by D1mon** [qax-os/excelize#1166]
before:
![before](https://user-images.githubusercontent.com/3872485/156323024-ece6489c-9dc5-4793-a8d9-8261bdc94a2a.png)
and file size: 53.3 KiB

after f.Save():
![after](https://user-images.githubusercontent.com/3872485/156323175-ede0aad9-e12b-40b8-8b27-d6ed22c740f4.png)
and file size: 43.7 KiB

**Steps to reproduce the issue:**
```
package main

import (
	"log"
	"github.com/xuri/excelize/v2"
)

func main() {
	f, err := excelize.OpenFile("1.xlsx")
	if err != nil {
		log.Fatal(err)
	}
	f.Save()
}
```

**Output of `go version`:**
go version go1.17.7 linux/amd64

**Excelize version or commit ID:**
2.5.0

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
Linux, onlyoffice-desktopeditors 7.0.0-1

[1.xlsx](https://github.com/qax-os/excelize/files/8167929/1.xlsx)

**Comment by xuri** [comment]
This issue duplicate with #1119, and #1130 and has been fixed on [`236ee61`](../commit/236ee61d201e45b1fe33a58b290adb7ee32d3488), please upgrade to the master branch code, and this patch will be released in the text version.

**Comment by xuri** [comment]
I will close this issue. If you still have any questions, please let me know.
